src/bindings/js/node/include/tensor.hpp

```diff
@@ -51,6 +51,9 @@ class TensorWrap : public Napi::ObjectWrap<TensorWrap> {
     Napi::Value get_shape(const Napi::CallbackInfo& info);
     /** @return Napi::String containing ov::element type. */
     Napi::Value get_element_type(const Napi::CallbackInfo& info);
+    /**@return Napi::Number containing tensor size*/
```

Comment:
```suggestion
    /**@return Napi::Number containing tensor size as total number of elements.*/
```

Reply:
@almilosz I have made the requested changes